community/civilized-discussion.md

```diff
@@ -0,0 +1,112 @@
+
+You may wish to respond by disagreeing. That’s fine. But remember to _criticize ideas, not people_. Please avoid:
+
+* Name-calling
+* Ad hominem attacks
+* Responding to a post’s tone instead of its actual content
+* Knee-jerk contradiction
+
+Instead, provide thoughtful insights that improve the conversation.
+
+Write your opinions **as opinions**. Rather than saying "everyone knows Python is the best language for beginners" or "Python is the best language for beginners", say "I feel that Python is the best language for beginners because ...". We take this seriously as it's an innocent looking habit that can be extremely disruptive to an online community. If you constantly write opinions as facts, we will ask you to stop and refer you to these guidelines. If you continue, we may choose to ban you.
+
+<a name="derailing"></a>
+
+## [Stay on topic - don’t derail](#derail)
+
+When posting, always think about whether you’re moving the conversation in the direction that the original poster intended, or whether you’re pulling it in a different direction. If it’s the latter, it's generally better to create a new topic.
+
+For example, if there’s a topic on cats, but you prefer dogs, don’t create a post saying “cats are rubbish - I like dogs” - instead create a new topic on dogs with your favorite puppy pictures in.
+
+<a name="participate"></a>
+
+## [Your Participation Counts](#participate)
+
+The conversations we have here set the tone for every new arrival. Help us influence the future of this community by choosing to engage in discussions that make this forum an interesting place to be &mdash; and avoiding those that do not.
+
+Discourse provides tools that enable the community to collectively identify the best (and worst) contributions: bookmarks, likes, flags, replies, edits, watching, muting and so forth. Use these tools to improve your own experience, and everyone else’s, too.
+
+Let’s leave our community better than we found it.
+
+<a name="flag-problems"></a>
+
+## [If You See a Problem, Flag It](#flag-problems)
+
+Moderators have special authority; they are responsible for this forum. But so are you. With your help, moderators can be community facilitators, not just janitors or police.
+
+When you see bad behavior, don’t reply. Replying encourages bad behavior by acknowledging it, consumes your energy, and wastes everyone’s time. _Just flag it_. If enough flags accrue, action will be taken, either automatically or by moderator intervention.
+
+In order to maintain our community, moderators reserve the right to remove any content and any user account for any reason at any time. Moderators do not preview new posts; the moderators and site operators take no responsibility for any content posted by the community.
+
+<a name="be-civil"></a>
+
+## [Always Be Civil](#be-civil)
+
+Nothing sabotages a healthy conversation like rudeness:
+
+* Be civil. Don’t post anything that a reasonable person would consider offensive, abusive, or hate speech.
```

Comment:
I feel like "reasonable person" is quite judgey here. People will have different ideas of what's offensive, abusive, or hate speech, and implying that someone whose idea of that might be different from the moderators is an "unreasonable person" is quite harsh. This text violates the guidelines above about criticising ideas, not people.

Reply:
I agree with what the point is trying to make and I agree with what Sascha is saying. [Point in case](https://www.npr.org/2020/09/29/917747123/you-literally-cant-believe-the-facts-tucker-carlson-tells-you-so-say-fox-s-lawye) (without a comment about this case, or the outcome, or the subjects, but just take it at face value that "what is reasonable" means very different things to very different people)